packages/ruby/lib/readme/http_response.rb

```diff
@@ -13,12 +13,12 @@ def self.from_parts(status, headers, body)
     def body
       if raw_body.respond_to?(:rewind)
         raw_body.rewind
-        content = raw_body.each.reduce('', :+)
+        content = raw_body.each.sum('')
```

Comment:
I think it's more conventional to use `#join` for strings, and the docs say its faster? 
https://ruby-doc.org/core-2.7.0/Array.html#method-i-sum

```suggestion
        content = raw_body.each.join('')
```

Reply:
I get a bunch of these when I try to run the tests after switching to join(), any ideas?

```ruby
Failure/Error: content = raw_body.each.join('')

NoMethodError:
  undefined method `join' for #<Enumerator: #<StringIO:0x00000001058d50f8>:each>

          content = raw_body.each.join('')
                                ^^^^^
```

Reply:
Oh, oops. It looks like the `body` isn't a regular `Enumerable`, so `each.sum` is kinda the only option. Nevermind!

Reply:
https://www.rubydoc.info/gems/rack/Rack%2FResponse:initialize